controllers/synapse/synapse_controller_test.go

```diff
@@ -1129,6 +1129,8 @@ var _ = Describe("Integration tests for the Synapse controller", Ordered, Label(
 
 					var createdSignaldDeployment *appsv1.Deployment
 					var createdSignaldPVC *corev1.PersistentVolumeClaim
+					var createdMautrixSignalServiceAccount *corev1.ServiceAccount
+					var createdMautrixSignalRoleBinding *rbacv1.RoleBinding
```

Comment:
Here and below on the code I see where some of the SA and Role, RoleBinding are going. On that topic I would highly encourage to let OLM do that part if possible. But I know that would require creating bundle etc... It's definitely not recommended letting the operator manage RBAC resources. We could let it there for now if it speeds up the development but I would point out some backlog issues to move this responsibility to OLM and lower down operator priviledges.

Reply:
I agree, and I have an issue for that already (there is also a SA + RB for the Synapse pods)

Added a TODO comment linking to https://github.com/opdev/synapse-operator/issues/19

Reply:
Cool!